external-import/mandiant/src/connector/indicators.py

```diff
@@ -144,20 +145,34 @@ def process(connector, indicator):
         # "name": "CAMP.22.049",
         # "title": "EMOTET Operations Resume in November 2022 Following Hiatus"
         # }
+
         # Add the campaign
-        stix_campaign = stix2.Campaign(
-            id=campaign["id"],
-            name=campaign["title"],
-            aliases=[campaign["name"]],
-            created_by_ref=connector.identity.get("standard_id"),
-            allow_custom=True,
-        )
-        items.append(stix_campaign)
-        items.append(
-            indicator_create_stix_relationship(
-                connector, stix_indicator, indicator, stix_campaign
+        # Allow import of Campaigns with related entities
+        if connector.import_full_campaigns:
+            bundle_campaign_with_rel = process_campaign(connector, campaign)
```

Comment:
Can you explain the with_rel naming?

Reply:
Yes, when processing campaigns here, we get the campaign and all details of the campaign but also its related entities which is not the case if import_indicators_with_full_campaigns is false

Reply:
rel_ stand for related_entities ?

Reply:
Yes, maybe my comment above is not explicit enough? Or do you have an idea of naming?

Reply:
You cant put the full name aka related_entities instead of rel.
No need for me to try to shorten the name is its not easy to understand :)